modules/client-handler/src/main/java/org/apache/ignite/client/handler/JdbcQueryEventHandlerImpl.java

```diff
@@ -82,42 +83,48 @@ public class JdbcQueryEventHandlerImpl extends JdbcHandlerBase implements JdbcQu
     /** {@link SqlQueryType}s allowed in JDBC update statements. **/
     private static final Set<SqlQueryType> UPDATE_STATEMENT_QUERIES = Set.of(DML, SqlQueryType.DDL, SqlQueryType.KILL);
 
+    /**
+     * Observation timestamp tracker.
+     * TODO: IGNITE-24053 Remove this after the issue will be fixed.
+     * */
+    private final HybridTimestampTracker timestampTracker = new HybridTimestampTracker();
```

Comment:
I've checked the mentioned JIRA, and it doesn't seem correct. At the moment, JDBC driver doesn't track time on client's side, and driver cannot connect to more than one server. During connection, server issues `connectionId` for every `java.sql.Connection` instance, and every request from the connection provides `connectionId`.

Given that said, we need to create one tracker per `JdbcConnectionContext` and use it for every request within this connection. Changes in JDBC-related requests processing methods are not necessary since driver doesn't track observable time

Reply:
Why does the JDBC driver cannot track an observation timestamp?

Reply:
because it doesn't need to. There is no support for multi-node connections yet (simultaneous connections to several servers within the same cluster), so why should we bother?

Reply:
I think it is not right.
We were not assumed to work with a client without an observation timestamp.